hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/store/driver/StateStoreDriver.java

```diff
@@ -35,6 +36,7 @@
  * provider. Driver implementations will extend this class and implement some of
  * the default methods.
  */
+@InterfaceAudience.Public
```

Comment:
Can you statically import the Public directly?
Not sure what is the agreement or standard these days for this.

Reply:
It is possible to statically import Public, but I think it makes the code less readable having just `@Public` vs `@InterfaceAudience.Public`. 

The oracle documentation also only recommends static imports when there is frequent access to the value.
https://docs.oracle.com/javase/7/docs/technotes/guides/language/static-import.html#:~:text=So%20when%20should%20you%20use,from%20one%20or%20two%20classes.